skimage/io/_io.py

```diff
@@ -130,6 +130,9 @@ def imsave(fname, arr, plugin=None, **plugin_args):
             plugin = 'tifffile'
     if is_low_contrast(arr):
         warn('%s is a low contrast image' % fname)
+    if (type(arr) == np.ndarray and arr.dtype == bool) or \
+        any(isinstance(ele, bool) for ele in arr):
```

Comment:
@alexandrejaguar thanks for this!
Anyhow, `any(isinstance(ele, bool) for ele in arr` doesn't work properly for 2D+ lists.

I suggest to cast the input `arr` to ndarray explicitly in the beginning of the function.